docs/website/docs/general-usage/incremental-loading.md

```diff
@@ -391,6 +392,62 @@ must be unique for a root table. We are working to allow `updated_at` style trac
 column in the root table to stamp changes in nested data.
 * `merge_key(s)` are (for now) ignored.
 
+### `upsert` strategy
+
+:::caution
+The `upsert` merge strategy is currently only supported for these destinations:
+- `postgres`
```

Comment:
you could consider mentioning this on the three destination pages.

Reply:
`upsert` will soon be supported for all destinations. I think it's better to leave this here for now and remove the `caution` block when generic support has been implemented.